src/test/java/com/epam/tc/hw2/ex1/TestForFirstExercise.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.tc.hw2.ex1;
+
+import static org.assertj.core.util.Lists.newArrayList;
+
+import com.epam.tc.hw2.SeleniumBaseClass;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+
+
+
+public class TestForFirstExercise extends SeleniumBaseClass {
+
+
+    @Test
+    public void testForFirstExercise() {
+
+        //Step 2. Assert Browser title
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(driver.getTitle()).isEqualTo("Home Page");
+
+        //Step 3. Perform login
+        WebElement findDropDown = driver.findElement(By.className("navbar-right"));
+        wait.until(ExpectedConditions.elementToBeClickable(findDropDown)).click();
+
+        WebElement username = driver.findElement(By.id("name"));
+        wait.until(ExpectedConditions.visibilityOf(username)).sendKeys("Roman");
+
+        WebElement password = driver.findElement(By.id("password"));
+        wait.until(ExpectedConditions.visibilityOf(password)).sendKeys("Jdi1234");
+
+        WebElement buttonEnter = driver.findElement(By.id("login-button"));
+        wait.until(ExpectedConditions.elementToBeClickable(buttonEnter)).click();
+
+        //4. Assert Username is loggined
+        WebElement logUsername = driver.findElement(By.id("user-name"));
+        wait.until(ExpectedConditions.visibilityOf(logUsername));
+        softly.assertThat(logUsername.getText()).isEqualTo("ROMAN IOVLEV");
+
+        //5. Assert that there are 4 items on the header section are displayed and they have proper text
+        List<WebElement> findHeaderItems = wait.until(ExpectedConditions.numberOfElementsToBeMoreThan(
+                By.cssSelector(".m-l8 > li"), 3));
+
+        List<String> expectedHeaderItems = newArrayList(
+                "HOME",
+                "CONTACT FORM",
+                "SERVICE",
+                "METALS & COLORS"
+        );
+
+        softly.assertThat(findHeaderItems)
+                .extracting(WebElement::getText)
+                .containsExactlyElementsOf(expectedHeaderItems);
+
+        //6. Assert that there are 4 images on the Index Page and they are displayed
+        List<WebElement> imageItems = wait.until(
+                ExpectedConditions.numberOfElementsToBeMoreThan(
+                By.cssSelector("div[class*=benefit] > span[class*=icons-benefit]"), 3));
+
+        softly.assertThat(imageItems).hasSize(4).allMatch(WebElement::isDisplayed);
+
+        //7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+        List<WebElement> textUnderImage = wait.until(
+                ExpectedConditions.numberOfElementsToBeMoreThan(By.className("benefit-txt"), 3));
+
+        List<String> expectedTextUnderImages = newArrayList(
```

Comment:
Can be replaced like
```suggestion
        List<String> expectedTextUnderImages = List.of(
```
This is not critical. Just for your information